g_octave/description.py

```diff
@@ -269,13 +269,12 @@ def __getattr__(self, name):
 
 class SvnDescription(Description):
```

Comment:
would it be too dificult to rename this to MercurialDescription, or something like that?